update a term (#21955)
mdn/content: files/en-us/web/css/visual_formatting_model/index.md

The string is split into three boxes in the box tree. The part of the string before the paragraph element is wrapped in an anonymous box, then we have the `<p>`, which generates a box, and then another anonymous box.

> Something to consider about these anonymous boxes is that they inherit styles from their direct parent, but you cannot change how they look by targeting the anonymous box. In my examples, I am using a direct child selector to target the children of the container. This does not change the anonymous boxes, as they are not an element of the parent block.

Reviewer: I think you're right. The `*` selector chooses all elements in the scope "by definition". So if that isn't finding an anonymous box, it's because the box isn't an element.

How about this? My reasoning is that he current wording says they are not elements of the parent block (true), which might imply though that they are elements of some other box.

```suggestion
Something to consider about these anonymous boxes is that they inherit styles from their direct parent, but you cannot change how they look by targeting the anonymous box. In my examples, I am using a direct child selector to target the children of the container. This does not change the anonymous boxes, as they are not "elements" as such.
```

@dipikabh Fly by comment - this is my interpretation of the spec, but you might know something I don't.
